src/HeaderBar/Apps.js

```diff
@@ -168,9 +168,10 @@ function List({ apps, filter }) {
     return (
         <div data-test="headerbar-apps-menu-list">
             {apps
-                .filter(({ displayName }) => {
+                .filter(({ displayName, name }) => {
+                    const appName = displayName || name;
```

Comment:
Moving this check up here means that we should be able to remove the check here: https://github.com/dhis2/ui-widgets/pull/274/files#diff-a8543f56f1a36f99128200467054b8d3R180

Reply:
Yeah, I repeated the check because I did not want to add a new map only to join the ``displayName || name``. Filtering a ReactNode is probably not a good idea.

Reply:
Alright, cool. I prefer duplicating the check (the current implementation) in that case. :+1: